libs/feature-flags/src/lib/features.ts

```diff
@@ -44,6 +44,7 @@ export enum Features {
   servicePortalEducationGraduation = 'isServicePortalEducationGratuationModuleEnabled',
   sessionHistory = 'sessionHistory',
   servicePortalConsentModule = 'isServicePortalConsentModuleEnabled',
+  servicePortalHealthRightsModule = 'isServicePortalHealthRightsModuleEnabled',
```

Comment:
This feature should also be used to block access to the graphql resolvers. Can use the FeatureFlag nest module and decorator.

Reply:
ah yeah, will do